tools/run_tests/python_utils/dockerjob.py

```diff
@@ -55,7 +55,7 @@ def docker_mapped_port(cid, port, timeout_seconds=15):
             output = subprocess.check_output(
                 "docker port %s %s" % (cid, port), stderr=_DEVNULL, shell=True
             ).decode()
-            return int(output.split(":", 2)[1])
```

Comment:
is this change really needed to pass on CI? Let's leave this alone if this is only needed for a manual run

Reply:
Interop tests are not run in PR checks, I can only manually run them.

This is needed for the case that docker container has both ipv4 and ipv6 stacks so `docker port` returns two lines of ip:port, this change alway uses the first line, which should work with single stack container as well.

But sure, let me remove it.

Reply:
Got it. I'd prefer to leave it out of this PR then since I'd prefer to not touch this python code here is we can avoid it.

But if the change is helpful, maybe it can go in its own PR (and assigned to @jtattermusch )